documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1,4 +1,4 @@
-[id="debezium-connector-for-sql-server"]
+DebeziumInstall[id="debezium-connector-for-sql-server"]
```

Comment:
This looks a bit odd. Won't this show up in rendered docs?

Reply:
Yes, sorry! That as unintentional paste and I caught the spot where I meant to paste that. 
Update in the momentary commit.